src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_validators.py

```diff
@@ -0,0 +1,19 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+import re
+
+from azure.cli.core._util import CLIError
+
+def validate_registry_name(namespace):
+    if namespace.registry_name:
+        registry_name = namespace.registry_name
+        if len(registry_name) < 5 or len(registry_name) > 60:
```

Comment:
>if len(registry_name) < 5 or len(registry_name) > 60: [](start = 8, length = 53)

The approach we take with the main modules is to avoid doing this kind of client-side validation provided the server returns a useful error. That way if the service changes the validation rules, the command package would not need to be revised. If the service returns a crappy error message (or worse, none) or the response time could be relatively long before surfacing an error, then we'd do this client-side.